backend/routes/forms/form.py

```diff
@@ -34,3 +35,20 @@ async def get(self, request: Request) -> JSONResponse:
             return JSONResponse(form.dict(admin=admin))
 
         return JSONResponse({"error": "not_found"}, status_code=404)
+
+    @requires(["authenticated", "admin"])
+    async def delete(self, request: Request) -> JSONResponse:
+        """Deletes form by ID."""
+        if not await request.state.db.forms.find_one(
+            {"_id": request.path_params["form_id"]}
+        ):
+            return JSONResponse({"error": "not_found"}, status_code=404)
+
+        await request.state.db.forms.delete_one(
+            {"_id": request.path_params["form_id"]}
+        )
+        await request.state.db.responses.delete_many(
+            {"form_id": request.path_params["form_id"]}
+        )
+
+        return JSONResponse(status_code=204)
```

Comment:
This will return the JSON literal `null` as the response body.

I think it would be better practice to return a `{"status": "okay"}` and 200 code or alternative return a normal response with empty body and 204 status.